nixos/modules/services/hardware/sane.nix

```diff
@@ -98,6 +102,34 @@ in
       '';
     };
 
+    hardware.sane.drivers.scanSnap = mkOption {
+      type = types.submodule ({ config, ... }: {
```

Comment:
Why are you using a submodule here?

Reply:
idk, brainfart. The module system is too complicated for my mouse brain.